File: Back/src/domain/Entities/Employees.py
import uuid

class Employees:
    id:int;
    firstName:str;
    lastName:str;
    code:str;
    def __init__(self, _id, _firstName, _lastName)->None:
        self.id = _id;
        self.firstName = _firstName;
        self.lastName = _lastName;

    async def generateCode(self):
        self.code = str(uuid.uuid4());

    async def GetCode(self):
        return self.code;

    async def SetCode(self, code:str):
        self.code = code;

File: BackEnd/Domain/Entities/Employees.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Employees
    {
        [Key]
        public int id { get; set; }
        public string firstName { get; set; }
        public string lastName { get; set; }
        private string code { get; set; } 

        public Employees()
        {

        }
        public async Task GenerateCode()
        {
            this.code = new Guid().ToString();
        }
        public string GetCode()
        {
            return this.code;
        }
        public void SetCode(string code)
        {
            this.code = code;
        }
        
    }
}
